### DISCUSSION TITLE
GitHub Projects on mobile

### DISCUSSION THREAD

**Discussion by thechnet** [community/community#3471] [Mobile] [answered]
The mobile app should include an interface for GitHub _Projects_.

**Comment by PranamBhat** [comment]
It's going to be a new feature @evi-liu @becca

**Comment by candyho** [comment, marked as answer]
Hey thanks for your feedback! Feel free to upvote on this topic related to Projects to help boost awareness of this request! - [#3023](https://github.com/github/feedback/discussions/3023)
Also appreciate if you can share more details with us in the thread above 👆 - how would you expect to manage projects on mobile? Do you expect to view the full kanban board on mobile? What are the key project features that you think we should cater on mobile?

**Reply by PranamBhat** [reply]
Perfect!

**Reply by erikr** [reply]
Would expect to view full Kanban board on mobile, be able to create and move issues, and be able to see labels and owners in the board view.
